tinytodo-go/README.md

```diff
@@ -24,4 +24,4 @@ See [TinyTodo's README](../tinytodo/README.md) for more information.
 
 ## Comparison with TinyTodo
 
-TinyTodo-Go is constrained by the features of [`cedar-go`](https://github.com/cedar-policy/cedar-go). Refer to [this README](https://github.com/cedar-policy/cedar-go?tab=readme-ov-file#comparison-to-the-rust-implementation) to learn about the missing features.
\ No newline at end of file
+TinyTodo-Go is constrained by the features of [`cedar-go`](https://github.com/cedar-policy/cedar-go). Refer to [this README](https://github.com/cedar-policy/cedar-go/tree/v0.3.2?tab=readme-ov-file#comparison-to-the-rust-implementation) to learn about the missing features.
```

Comment:
Can you add a note somewhere in the README that says that this example uses v0.3.2?

Reply:
addressed in 7e631e66ea59aa9b7ee84450191eb17b18c4b4cd - please see if the note is acceptable